src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/upgrade/upgrade.component.html

```diff
@@ -47,13 +47,7 @@ <h5>{{ version }}</h5>
              id="clusterStatus">
       <div class="d-flex flex-column justify-content-center align-items-center">
         <ng-template #healthChecks>
-          <ul>
-            <li *ngFor="let check of healthData.health.checks">
-              <span [ngStyle]="check.severity | healthColor"
-                    [class.health-warn-description]="check.severity === 'HEALTH_WARN'">
-              {{ check.type }}</span>: {{ check.summary.message }}
-            </li>
-          </ul>
+          <cd-health-checks [healthData]="healthData"></cd-health-checks>
```

Comment:
Is there a need to pass the entire healthData to the other component?
```suggestion
          <cd-health-checks [healthData]="healthData.health.check"
                            *ngIf="healthData?.health?.check"></cd-health-checks>
```